components/cli/pkg/commands/run.go

```diff
@@ -1042,6 +995,47 @@ func getYamlFiles(path string) ([]string, error) {
 	return files, nil
 }
 
+// extractImage extracts the image into a temporary directory and returns the path.
+// Cleaning the path after finishing your work is your responsibility.
+func ExtractImage(cellImage *util.CellImage, spinner *util.Spinner) (string, error) {
```

Comment:
Suggestion: CellImage struct can probably be moved to a different package, say `cell` package.

Reply:
Noted, shall we move cellImage struct to image package?

Reply:
This change introduces a circular dependency. Lets complete this as a part of the CLI refactoring effort.

Reply:
Ack.